frontend/src/lib/components/icons.tsx

```diff
@@ -1309,12 +1309,12 @@ export function EventStackGearIcon({ width = 24, height = 24, ...props }: React.
     )
 }
 
-export function LiveIcon({ width, height, ...props }: React.SVGProps<SVGSVGElement>): JSX.Element {
+export function LiveIcon({ width = 24, height = 24, ...props }: React.SVGProps<SVGSVGElement>): JSX.Element {
     return (
         <svg
             width={width}
             height={height}
-            viewBox="0 0 20 16"
+            viewBox="0 0 24 24"
```

Comment:
```suggestion
            viewBox="0 0 20 16"
```

Reply:
All the icons are based on a 24px viewbox. Part of the system of icons being consistent is using a consistent viewbox sizing. Why would we change this for each icon?

Reply:
Unresolved so @alexkim205 can answer

Reply:
I think it's because the height and width of the item disagree with the viewbox parameters. It could also be the specific SVG file in use. Here's a fresh export at the proper viewbox size. My concern with variable size viewboxes is that icons will layout different based on context. For them to be repeatable and easily aligned, standardized dimensions and placement are important. Happy to defer, just surprised to see this as a variable parameter. 

![live-events](https://user-images.githubusercontent.com/254612/159991207-02c02431-04f9-4608-abef-cc55e4f2a90f.svg)

Reply:
Yup you're exactly right, I think the svg we're using here isn't sized 24 x 24 which means that if we set a default height and width to be 24 here, the icon will be offset by the difference. I'll go ahead and replace this svg w/ one that takes up the full 24 x 24